scripts/Synapse/updateECRDataStoreIncidentID.ipynb

```diff
@@ -50,28 +49,28 @@
-        "    .when((lower(ecr.test_type_19).isin(covid_test_types) & lower(ecr.test_result_19).isin(covid_positive_results) & lower(ecr.conditions).isin(covid_conditions)), ecr.specimen_collection_date_19)\n",
-        "    .when((lower(ecr.test_type_20).isin(covid_test_types) & lower(ecr.test_result_20).isin(covid_positive_results) & lower(ecr.conditions).isin(covid_conditions)), ecr.specimen_collection_date_20)\n",
+        "    when((lower(ecr.test_type_1).isin(covid_test_types) & lower(ecr.test_result_1).isin(covid_positive_results)), ecr.specimen_collection_date_1)\n",
+        "    .when((lower(ecr.test_type_2).isin(covid_test_types) & lower(ecr.test_result_2).isin(covid_positive_results)), ecr.specimen_collection_date_2)\n",
+        "    .when((lower(ecr.test_type_3).isin(covid_test_types) & lower(ecr.test_result_3).isin(covid_positive_results)), ecr.specimen_collection_date_3)\n",
+        "    .when((lower(ecr.test_type_4).isin(covid_test_types) & lower(ecr.test_result_4).isin(covid_positive_results)), ecr.specimen_collection_date_4)\n",
+        "    .when((lower(ecr.test_type_5).isin(covid_test_types) & lower(ecr.test_result_5).isin(covid_positive_results)), ecr.specimen_collection_date_5)\n",
+        "    .when((lower(ecr.test_type_6).isin(covid_test_types) & lower(ecr.test_result_6).isin(covid_positive_results)), ecr.specimen_collection_date_6)\n",
+        "    .when((lower(ecr.test_type_7).isin(covid_test_types) & lower(ecr.test_result_7).isin(covid_positive_results)), ecr.specimen_collection_date_7)\n",
+        "    .when((lower(ecr.test_type_8).isin(covid_test_types) & lower(ecr.test_result_8).isin(covid_positive_results)), ecr.specimen_collection_date_8)\n",
+        "    .when((lower(ecr.test_type_9).isin(covid_test_types) & lower(ecr.test_result_9).isin(covid_positive_results)), ecr.specimen_collection_date_9)\n",
+        "    .when((lower(ecr.test_type_10).isin(covid_test_types) & lower(ecr.test_result_10).isin(covid_positive_results)), ecr.specimen_collection_date_10)\n",
+        "    .when((lower(ecr.test_type_11).isin(covid_test_types) & lower(ecr.test_result_11).isin(covid_positive_results)), ecr.specimen_collection_date_11)\n",
+        "    .when((lower(ecr.test_type_12).isin(covid_test_types) & lower(ecr.test_result_12).isin(covid_positive_results)), ecr.specimen_collection_date_12)\n",
+        "    .when((lower(ecr.test_type_12).isin(covid_test_types) & lower(ecr.test_result_13).isin(covid_positive_results)), ecr.specimen_collection_date_13)\n",
+        "    .when((lower(ecr.test_type_14).isin(covid_test_types) & lower(ecr.test_result_14).isin(covid_positive_results)), ecr.specimen_collection_date_14)\n",
+        "    .when((lower(ecr.test_type_15).isin(covid_test_types) & lower(ecr.test_result_15).isin(covid_positive_results)), ecr.specimen_collection_date_15)\n",
```

Comment:
Did we want to use the new column that has the actual loinc codes in it and then also update the covid_test_types to contain the LOINC codes?  I have a PR up for the newly added test_type_code_# column for each lab result which will make the LOINC codes accessible.  Nice work BTW!

Reply:
Good call! I made the change here and will also change in the extract TSV job as well.